_includes/cloud/backup-db.md

```diff
@@ -8,6 +8,9 @@ We recommend creating a backup of your project before an upgrade. Use the follow
     magento-cloud db:dump
     ```
 
+    {: .bs-callout-info }
+    `magento-cloud db:dump` command does not lock tables within creating dump operation because it runs [mysqldump](https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html) with a `--single-transaction` flag. 
```

Comment:
```suggestion
    The `magento-cloud db:dump` command runs the [mysqldump](https://dev.mysql.com/doc/refman/8.0/en/mysqldump.html) command with the `--single-transaction` flag, which  allows you to back up your database without locking the tables.
```

Reply:
**implemented**

Thank you, @meker12 !